apis/database.graphql

```diff
@@ -918,10 +954,28 @@ type OpticalData implements Node & Data {
   approval: ResponseApproval!
 
   """
-  Mirrored nearnormal hemispherical visible transmittance values that occur in
-  the data
+  Mirrored nearnormal hemispherical visible transmittance values which can be found in the data set. If it can only be calculated with the data set, then nearnormalHemisphericalVisibleTransmittances must not be returned.
```

Comment:
Here and below I would rephrase `If it can only be calculated with the data set, then nearnormalHemisphericalVisibleTransmittances must not be returned.` to `If some or all of those values do not occur verbatim in the data, they need not occur in the list, even if they can be calculated based on the data.`